package.json

```diff
@@ -33,12 +34,22 @@
             "make_targets": {
                 "darwin": [
                     "zip"
+                ],
+                "win32": [
+                    "squirrel"
                 ]
             },
             "electronPackagerConfig": {
                 "packageManager": "npm",
                 "icon": "assets/icon.icns"
             },
+            "electronWinstallerConfig": {
```

Comment:
Need to add [configuration for windows](https://github.com/electron/windows-installer#usage).